models/cfp.py

```diff
@@ -206,6 +208,16 @@
 MANUAL_REVIEW_TYPES = ["youthworkshop", "performance", "installation"]
 
 
+# This is a function rather than a constant so we can lean on the configuration
+# for event start & end times rather than hard coding stuff
+def get_days_map():
+    event_days = [
+        (event_start() + timedelta(days=day_idx))
```

Comment:
I think this should explicitly call .date(), as event_start is currently 11am and we add hours on below